src/scripts/libraryBrowser.js

```diff
@@ -86,11 +86,9 @@ export function getQueryPagingHtml (options) {
 
     html += '<div class="listPaging">';
 
-    if (showControls) {
-        html += '<span style="vertical-align:middle;">';
-        html += globalize.translate('ListPaging', totalRecordCount ? startIndex + 1 : 0, recordsEnd, totalRecordCount);
-        html += '</span>';
-    }
+    html += '<span style="vertical-align:middle;">';
+    html += globalize.translate('ListPaging', totalRecordCount ? startIndex + 1 : 0, recordsEnd || totalRecordCount, totalRecordCount);
```

Comment:
I think I would prefer we handle this when we assign the value of `recordsEnd` to reduce the logic here in the templating code.

```js
const recordsEnd = limit ? Math.min(startIndex + limit, totalRecordCount) : totalRecordCount;
```

Reply:
Will do.
Do you want to create a const for the `recordsStart` as well?
Then we can even make it more simple in the templating logic:
`const recordsStart = totalRecordCount ? startIndex + 1 : 0;`

Reply:
Sure that sounds good.

Reply:
Done